system.c

```diff
@@ -49,6 +50,25 @@ void system_init()
 
   OCR2A = 249;          //(249+1) * 64 prescale / 16Mhz = 1 ms
   OCR2B = 0;
+
+  /* KEYME SPECIFIC START */
+  // Setup Timer1 for AutoTriggering
+  // Timer 1 was used for the ADC interrupt. Reasoning for commenting this out is discussed
+  // below where the ADC ISR is defined.
+/*  PRR0 &= ~(1<<PRTIM1); // Gives power to Timer 1
+  TCCR1B &= ~(1<<WGM13); // waveform generation = 0100 = CTC
+  TCCR1B |=  (1<<WGM12);
+  TCCR1B |= (1<<CS11)|(1<<CS10); // Prescaler 64x
+  TCCR1A &= ~((1<<WGM11) | (1<<WGM10));
+  TCCR1A &= ~((1<<COM1A1) | (1<<COM1A0) | (1<<COM1B1) | (1<<COM1B0)); // Disconnect OC4 output
+
+  // Timer Match value of 0xFFFF. This can be reduced to retrieve voltage values sooner
+  OCR1A = 0XF0;
+  OCR1B = 0X00;
+
+  // Initialize ADC for Voltage Monitoring
+  init_ADC();*/
+  /* KEYME SPECIFIC END */
```

Comment:
You have added a whole bunch of commented-out code here. Either remove it or uncomment it (or explain why this isn't currently used but will be uncommented in the near future)